src/foraging_gui/Foraging.py

```diff
@@ -4649,14 +4652,12 @@ def start_gui_log_file(box_number):
     # Start the log file
     print('Starting a GUI log file at: ')
     print(logging_filename)
-    logging.basicConfig(
-        format=log_format,
-        level=logging.INFO,
-        datefmt=log_datefmt,
-        handlers=[
-            logging.FileHandler(logging_filename),
-        ]
-    )
+
+    log_formatter = logging.Formatter(fmt=log_format, datefmt=log_datefmt)
```

Comment:
You don't set the level as logging.INFO, is that the default? I would probably set it explicitly

Reply:
maybe you are just inheriting the level from the existing logger? In that case I would absolutely set the level

Reply:
Oh yeah I forgot to set the level I'll add that in